src/components/cc-global-footer.vue

```diff
@@ -129,19 +129,19 @@
                 <p class="caption">
                   Except where otherwise
                   <a
-                    href="https://creativecommons.org/policies#license"
+                    href="/policies/#license"
```

Comment:
I believe we may want to use the global footer on subdomains of creative commons.org. 

In the subdomain use-case, the root-level link might point to a non-existent route. So, it is probably safer to use the full URL,  including creativecommons.org domain for these types of links.

Reply:
I will merge this pull request since it is a genuine bug fix.

That said, I'm afraid I have to disagree with the approach of using relative URLs for all global-footer links because

- it breaks precedent with other CC sites, which use complete URLs for footer links (including to the licenses)
- in a development environment, the relative links may lead to non-existent paths
- our primary use-case should be production-oriented, and any concessions for development environments should be made secondarily (such as adding an optional `use-absolute-paths` property that would work in development and production)